source/admin_manual/list_of_events.rst

```diff
@@ -854,579 +487,380 @@ These events apply only for connections using the ``connection API``.
-+---------------------+------------------------------------------------------+
-| bytes_out           | Amount of data written, in bytes                     |
-+---------------------+------------------------------------------------------+
-| reason              | Disconnection reason                                 |
-+---------------------+------------------------------------------------------+
-
-
-server_connection_connected
----------------------------
-
-Emitted when an outgoing server connection was either successfully established
-or failed. Currently it is not possible to know which one happened.
-
-server_connection_disconnected
-------------------------------
-
-Emitted when a server connection is terminated.
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| bytes_in            | Amount of data read, in bytes                        |
-+---------------------+------------------------------------------------------+
-| bytes_out           | Amount of data written, in bytes                     |
-+---------------------+------------------------------------------------------+
-| reason              | Disconnection reason                                 |
-+---------------------+------------------------------------------------------+
 
 FS
 ==
 
-.. _event_fs:
+.. dovecot_core:event:: fs
 
-fs
---
+   May be inherited from various different parents (e.g. "Mail User" event)
+   or even from no parent.
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from environment (e.g. :ref:`event_mail_user`)                    |
-+---------------------+------------------------------------------------------+
 
-.. _event_fs_file:
+.. dovecot_core:event:: fs_file
 
-fs_file
--------
+   Inherits from fs or any other specified event (e.g. mail).
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from :ref:`event_fs` or any other specified event                 |
-| (e.g. :ref:`event_mail`)                                                   |
-+---------------------+------------------------------------------------------+
 
-.. _event_fs_iter:
+.. dovecot_core:event:: fs_iter
 
-fs_file
--------
+   :field file_type: * ``mail``: Email file
+                     * ``index``: Index bundle
+                     * ``box``: Mailbox directory (for creating/deleting it,
+                       if used by the storage driver)
+                     * ``fts``: FTS file
```

Comment:
There is no indication that the file_type field exists only for files accessed via obox.